### PR TITLE
"send_logs_level" bugfix

### PR DESCRIPTION
**Pull request by nikitamikhaylov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

For changelog. Remove if this is non-significant change.

Category (leave one):
- Bug Fix

Short description (up to few sentences):
Now client could receive all logs he wants.

 #5041

### CODE REVIEW

File: dbms/programs/server/TCPHandler.cpp [hunk 1]
```diff
@@ -170,8 +171,9 @@ void TCPHandler::runImpl()
             send_exception_with_stack_trace = query_context->getSettingsRef().calculate_text_stack_trace;
 
             /// Should we send internal logs to client?
+            CLIENT_LOGS_LEVEL = query_context->getSettingsRef().send_logs_level.value;
```
Comment (alexey-milovidov) [inline dbms/programs/server/TCPHandler.cpp]: Race condition when multiple concurrent queries are run: each will overwrite logs level.

**Review by alexey-milovidov — CHANGES_REQUESTED** [review]
.

File: dbms/programs/server/TCPHandler.cpp (hunk 1 above)
Comment (alexey-milovidov) [reply dbms/programs/server/TCPHandler.cpp]: Look at ThreadsStatus, ThreadStatusExt.

Comment (nikitamikhaylov) [reply dbms/programs/server/TCPHandler.cpp]: Race condition on atomic variable?

Comment (alexey-milovidov) [reply dbms/programs/server/TCPHandler.cpp]: Yes.

Comment (alexey-milovidov) [reply dbms/programs/server/TCPHandler.cpp]: Now you have to make .sh test that runs two queries in parallel.

**Pull request closed without merging**